homeassistant/components/ecobee/strings.json

```diff
@@ -37,6 +37,9 @@
       },
       "ventilator_min_type_away": {
         "name": "Ventilator min time away"
+      },
+      "aux_heat_cutover": {
+        "name": "Aux Heat Cutover Threshold"
```

Comment:
Please have the name in Sentence case, and should we write out `Aux`?

Reply:
https://github.com/home-assistant/core/pull/129474/commits/e6f240f7a424c4b96897eb4443056d502982bb52